cmd/openshift-tests/e2e.go

```diff
@@ -120,8 +120,8 @@ var staticSuites = testSuites{
 				}
 				return strings.Contains(name, "[Suite:openshift/conformance/serial") || isStandardEarlyOrLateTest(name)
 			},
-			// doubling default test timeout (15 mins) which is needed for external ip configuration related serial tests.
-			TestTimeout:         30 * time.Minute,
+			// etcd's vertical scaling test is expensive
```

Comment:
shouldn't we move that test to entirely new suite in that case?

Reply:
nope, we want to have at least one test in the serial suite to cover a wide range of various platforms and block on promotions.

Reply:
we should create a new suite for more scaling e2e tests